.github/workflows/markdown-link-check.yml

```diff
@@ -14,5 +14,7 @@ jobs:
       with:
         use-quiet-mode: 'yes'
         use-verbose-mode: 'no'
+        folder-path: './examples, ./docs/book'
```

Comment:
don't we need to check src/ as well? We might have log messages or similar that point to old links. Or can't we detect those anyways?

Reply:
Absolutely! This tool now only checks markdown files. So if the links are in a `.py` file it will miss it. But I noted there are also a couple of `.md` files in src/ 

I will add the src/ folder in :)